src/progress/ProgressBar.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { cx, Progress as RenderlesskitProgress } from "@renderlesskit/react";
+
+import { BoxProps } from "../box";
+import { useTheme } from "../theme";
+import { useProgress } from "./Progress";
+import { forwardRefWithAs } from "../utils/types";
+
+export type ProgressBarProps = BoxProps & {};
+
+export const ProgressBar = forwardRefWithAs<
+  ProgressBarProps,
+  HTMLDivElement,
+  "div"
+>((props, ref) => {
+  const { className, ...rest } = props;
+  const theme = useTheme();
+  const { state } = useProgress();
+  const { percent, isIndeterminate } = state;
+
+  return (
+    <RenderlesskitProgress
+      ref={ref}
+      aria-label="progress"
```

Comment:
🤔 should we actually provide the aria-label? or users should be able to provide that too?

Reply:
They can provide them it will just get overridden.
I think we should have the aria's within our UI components.